include/fastpath/fastpath.h

```diff
@@ -114,5 +119,41 @@ static inline reply_t *thread_state_get_replyObject_np(thread_state_t ts)
 }
 #endif
 
+static inline void cancelSignal_fp(tcb_t *tptr, notification_t *ntfnPtr)
+{
```

Comment:
Functions like these shouldn't be on the fastpath, it should bail out to the slowpath if a call like this is necessary.

Reply:
Sorry , that was a leftover from Shane's implementation that I didn't end up using.

Reply:
That is a relief :-)